cfgrammar/src/lib/yacc/grammar.rs

```diff
@@ -273,7 +295,7 @@ impl YaccGrammar {
             }
         }
 
-        YaccGrammar{
+        Ok(YaccGrammar{
```

Comment:
Is there a reason for not calling `YaccGrammar::new` here?

Reply:
This is basically `YaccGrammar::new` (well, `YaccGrammar::new_with_storaget`, but same difference), so we don't an infinitely recursive call here :)

Reply:
OK